Modules/xSharePoint/DSCResources/MSFT_xSPAppCatalog/MSFT_xSPAppCatalog.schema.mof

```diff
@@ -0,0 +1,24 @@
+/*
+**Description**
+
+This resource will configure the App Catalog settings for Web Applications and Host Named Site Collections(HNSC).
```

Comment:
Not sure this is specific to HNSC's - just a site collection in a web app. It can still be path based site collections and work just fine as well